autofocus/predict/app/models/ZipArchive.py

```diff
@@ -0,0 +1,88 @@
+import os
+from zipfile import ZipFile
+
+from .File import File
+from ..requests.Validator import ALLOWED_IMAGE_FILES
+from ..utils import allowed_file
+
+
+class ZipArchive:
```

Comment:
I don't think we need our own ZipArchive class -- the standard library has these capabilities https://docs.python.org/3/library/zipfile.html

Reply:
Here the same as for the 'File' (now 'TemporaryFile'). It adds more functionality to the build in functions and can be reused.